gpsoauth/__init__.py

```diff
@@ -16,9 +18,50 @@
 auth_url = 'https://android.clients.google.com/auth'
 useragent = 'gpsoauth/' + __version__
 
+# Certain ciphers cause Google to return 403 Bad Authentication.
+CIPHERS = ":".join(
+    [
+        "ECDHE+AESGCM",
+        "ECDHE+CHACHA20",
+        "DHE+AESGCM",
+        "DHE+CHACHA20",
+        "ECDH+AES",
+        "DH+AES",
+        "RSA+AESGCM",
+        "RSA+AES",
+        "!aNULL",
+        "!eNULL",
+        "!MD5",
+        "!DSS",
+    ]
+)
```

Comment:
this was pulled from [here](https://github.com/urllib3/urllib3/blob/e47d3e7456b95c59ba85c55b6af0c898b63e5c4f/src/urllib3/util/ssl_.py#L64-L99), yeah? Would it be possible to build our list from theirs and just maintain a blocklist on our end for the known troublesome ciphers? I'm worried about having to maintain this list.

Reply:
Yes, that's the correct source. I had to remove the last entry `!AESCCM` which was added recently to make it work again. The rest of the list can stay the same for now.

Reply:
so  something like
```
   context.set_ciphers(cipher for cipher in urllib3.util.ssl_.DEFAULT_CIPHERS if cipher != "!AESCCM")
```
should also work.

Reply:
let me know if I should test & update this PR

Reply:
great; that sounds good to me 👍